### PR TITLE
Invite fixes

### PR DESCRIPTION
**Pull request by DavidS-om**

### CODE REVIEW

File: cmd/invite.go [hunk 1]
```diff
@@ -49,6 +56,13 @@ var listCmd = &cobra.Command{
...
 	Short: "Create a new invite",
+	PreRun: func(cmd *cobra.Command, args []string) {
+		// Bind these to viper
+		err := viper.BindPFlags(cmd.Flags())
+		if err != nil {
+			log.WithError(err).Fatal("could not bind `manual-change` flags")
+		}
+	},
 	Run: func(cmd *cobra.Command, args []string) {
 		// Bind flags to viper
```
Comment (dylanratcliffe) [inline cmd/invite.go:67]: Let's remove these so we're not binding twice

File: cmd/root.go [hunk 2]
```diff
@@ -186,12 +186,8 @@ func ensureToken(ctx context.Context, requiredScopes []string) (context.Context,
 	}
 
 	// Check to see if the URL is secure
-	gatewayUrl := viper.GetString("gateway-url")
```
Comment (dylanratcliffe) [inline cmd/root.go:189]: Does the removal of this `gateway-url` flag mean it's disused? If so should we remove it from the CLI flags too so it doesn't show in the help

**Review by dylanratcliffe — CHANGES_REQUESTED** [review]
Couple comments

File: cmd/root.go (hunk 2 above)
Comment (DavidS-om) [reply cmd/root.go:189]: --gateway-url is used elsewhere, so it needs to stay.

**Review by dylanratcliffe — APPROVED** [review]

**Pull request merged**